data/i18n/zh-cn/zh-cn.toml

```diff
@@ -167,12 +172,6 @@ other = """Linux 基金会&reg;。保留所有权利。Linux 基金会已注册
 [main_documentation_license]
 other = """The Kubernetes 作者 | 文档发布基于 <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a> 授权许可"""
 
-[main_edit_this_page]
-other = "修改本页面"
-
-[main_github_create_an_issue]
-other = "报告 GitHub 问题"
-
```

Comment:
are you sure these can be removed?

Reply:
Yeah, please see the updated PR description ([here](https://github.com/kubernetes/website/pull/33758#issue-1238040775))